rllib/utils/spaces/space_utils.py

```diff
@@ -369,9 +369,12 @@ def map_(elem, s):
             elif s.dtype != elem.dtype:
                 elem = elem.astype(s.dtype)
 
-        elif isinstance(s, int):
+        elif isinstance(s, int) or isinstance(s, np.int_):
```

Comment:
Can you add a comment here on why we need to check for np.int_ and operate on np.float_?

Reply:
Done :)